compiler/sem/lambdalifting.nim

```diff
@@ -194,17 +194,13 @@ proc addHiddenParam(routine: PSym, param: PSym) =
 proc getHiddenParam(g: ModuleGraph; routine: PSym): PSym =
   let params = routine.ast[paramsPos]
   let hidden = lastSon(params)
-  if hidden.kind == nkSym and hidden.sym.kind == skParam and hidden.sym.name.s == paramName:
-    result = hidden.sym
-    assert sfFromGeneric in result.flags
-  else:
-    # writeStackTrace()
-    internalError(
-      g.config,
-      routine.info,
-      "internal error: could not find env param for " & routine.name.s)
 
-    result = routine
+  g.config.internalAssert(
+    hidden.kind == nkSym and hidden.sym.kind == skParam and hidden.sym.name.s == paramName,
+    routine.info, "internal error: could not find env param for " & routine.name.s)
```

Comment:
If you're going for one param per line, go all in.

```suggestion
    routine.info,
    "internal error: could not find env param for " & routine.name.s
  )
```

Reply:
Stretches it over multiple lines unneccessarily IMO, since neither the info nor the error message are particularily important or informative to the intent of the code.

Reply:
It makes it easier to separate the parameter from being "part of the string" for me, but it's not required.

Reply:
That's fair, I'd argue they are both only related to what is being output if the assert fails, so them being close together isn't an issue.